src/main/resources/archetype-resources/README.md

```diff
@@ -72,3 +72,48 @@ $symbol_pound$symbol_pound Deploying to Bluemix
     kubectl get -w pod ${rootArtifactId}-0
     ```
     Press control-C to exit once this prints a line with "1/1" and "Running".
+    
+** Your room is up!** 
+
+Do a quick verification: http://169.46.81.162/${rootArtifactId}
+
+$symbol_pound$symbol_pound Register your room
+
+1.  Go to [Game On!](https://gameontext.org) and sign in.
+
+2.  Click on the building icon in the top right of the game screen to go to the Room Management page.
+
+3.  Make sure **Create a new room** is selected from the **Select a room** drop-down.
+
+4.  Provide a descriptive title for your room, e.g. `The Shortest Cut`
+
+5.  A short nickname will be generated, but you can change it if you like (e.g. use your lab id: `${rootArtifactId}`), but remember what it is!
+
+6.  Describe your room (optional). The description provided here is used by the interactive map and other lists or indexes of defined rooms. The decription seen in the game will come from your code.
+
+7. The repository field is optional. Come back and fill it in if you decide to push this into a public repository. 
+
+8.  Specify the http endpoint as a basic health endpoint: `http://169.46.81.162/${rootArtifactId}`
+
+8.  Use a WebSocket URL for the WebSocket endpoint: `ws://169.46.81.162/${rootArtifactId}`
+
+9. Leave the token blank for now. That is an [Advanced adventure]() for another time.
+
+10. Describe the doors to your room (Optional). Describe each door as seen [from the outside](https://book.gameontext.org/walkthroughs/registerRoom.html#doors)
+    
+8.  Click **Register** to register the room and add it to the Map!
```

Comment:
This is out of sequence. I know the numbers don't really matter in markdown, but for readability's sake it is nicer to keep them in order (also because participants might be reading this README in the source code).